src/core/gui/XournalView.cpp

```diff
@@ -139,36 +143,40 @@ auto XournalView::onKeyPressEvent(GdkEventKey* event) -> bool {
         }
     }
 
+    // TODO: With GTK 4 there is no GdkEventKey. Use a GdkEventControllerKey and connect to its key-pressed signal
```

Comment:
For consistency: I think most of the GTK4 indications are written this way
```suggestion
    // Todo(gtk4) there is no GdkEventKey. Use a GdkEventControllerKey and connect to its key-pressed signal
```

Reply:
Thanks for the review! I changed the comment like suggested.

Merging in 24 hours if no objections are raised.